src/vendor/google.golang.org/grpc/internal/transport/controlbuf.go

```diff
@@ -832,7 +832,7 @@ func (l *loopyWriter) processData() (bool, error) {
 	dataItem := str.itl.peek().(*dataFrame) // Peek at the first data item this stream.
 	// A data item is represented by a dataFrame, since it later translates into
 	// multiple HTTP2 data frames.
-	// Every dataFrame has two buffers; h that keeps grpc-message header and d that is acutal data.
+	// Every dataFrame has two buffers; h that keeps grpc-message header and d that is actual data.
```

Comment:
no need to change fileds in vendor folders

Reply:
Oops, I try to avoid vendor folders